tests/bash/comp-tests.bash

```diff
@@ -154,6 +154,26 @@ _completionTests_verifyCompletion "testprog --customComp f" "firstComp forthComp
 _completionTests_verifyCompletion "testprog --customComp=" "firstComp secondComp forthComp" nofile
 _completionTests_verifyCompletion "testprog --customComp=f" "firstComp forthComp" nofile
 
+#################################################
+# Special characters
+#################################################
+if [ "$BASHCOMP_VERSION" = bash2 ]; then
+   (
```

Comment:
Can you remove the sub-shell so that these these failures get reported globally?